frontend/src/app/pages/content-management/added-product/added-product.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { ButtonLabel } from 'src/app/enums/button-label-enum';
+import { Product } from 'src/app/interfaces/product';
+import { ProductService } from 'src/app/services/product/product.service';
+
+@Component({
+  selector: 'app-added-product',
+  templateUrl: './added-product.component.html',
+  styleUrls: ['./added-product.component.scss']
+})
+export class AddedProductComponent {
+  public addingProductButtonText: string = ButtonLabel.ADD_PRODUCT;
+  public saveProductButtonText: string = ButtonLabel.SAVE_CHANGES;
+  public isAddProduct: Boolean = false;
+  public newProduct: Product = {
+    id: 0,
+    categoryId: 0,
```

Comment:
User which create product can't know anythign about categoryId. User knows only names of categories and when user create product he should type name of category instead of categoryId